packages/kbn-field-utils/src/utils/get_field_type_description.ts

```diff
@@ -33,110 +33,110 @@ export function getFieldTypeDescription(type?: string) {
   const knownType: KNOWN_FIELD_TYPES = type as KNOWN_FIELD_TYPES;
   switch (knownType) {
     case KNOWN_FIELD_TYPES.DOCUMENT:
-      return i18n.translate('unifiedFieldList.fieldNameDescription.recordField', {
+      return i18n.translate('fieldUtils.fieldNameDescription.recordField', {
```

Comment:
Should the id renaming be changed in the translation files as well? Since it's just the id that changed and not content. Wonder if the current i18n tooling -fix will sort correctly for us if we update the id in the translation files.

Reply:
Hi @qn895,
I restored translations in json files and rather replaced ids there https://github.com/elastic/kibana/pull/167179/commits/79dfae8cd8527839c71c5f7fcbd6b3709a93dbb8